swedbank-pay-design-guide-jekyll-theme.gemspec

```diff
@@ -1,22 +1,31 @@
 # frozen_string_literal: true
 
+gem_version = "undefined"
```

Comment:
Codacy found an issue: [Useless assignment to variable - `gem_version`.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5853053)